packages/react/src/primitives/Tabs/TabsPanel.tsx

```diff
@@ -8,21 +8,23 @@ import { View } from '../View';
 import { BaseTabsPanelProps, TabsPanelProps } from './types';
 import { primitiveWithForwardRef } from '../utils/primitiveWithForwardRef';
 import { TabsContext } from './TabsContext';
+import { useStableId } from '../utils/useStableId';
 
 const TabPanelPrimitive: Primitive<TabsPanelProps, 'div'> = (
   { className, value, children, role = 'tabpanel', ...rest },
   ref
 ) => {
   const { activeTab, isLazy } = React.useContext(TabsContext);
+  const idValue = useStableId();
 
   if (isLazy && activeTab !== value) return null;
 
   return (
     <View
       {...rest}
       role={role}
-      id={`${value}-panel`}
-      aria-labelledby={`${value}-tab`}
+      id={`${idValue}-panel`}
+      aria-labelledby={`${idValue}-tab`}
```

Comment:
If we are using `useStableId` in 2 different components, wouldn't that generate 2 different IDs? We need `aria-labelledby` in this component to be the `id` of the `TabItem` and vice-versa with `aria-controls` and this `id`

Reply:
oh, thats true

Reply:
I switched it back to using the given value, with spaces replaced with `-` to ensure the all of the labels work cohesively. This relies on the user giving each tab a unique value.

Reply:
Can we think of another solution to providing a unique id? Maybe passing a `React.useId` or `useStableId` down through context so it is shared? @zchenwei any thoughts on that?

I think it creates sort of a "hidden" requirement that each value has to be unique because it is also happens to be used to generate the ID. It also creates a scenario where the developer has to track down the cause of duplicate IDs or potentially inaccessible markup. This should be something we can safely back into the component. An example is if you inspect that markup and usage of the Radix Tabs component https://www.radix-ui.com/primitives/docs/components/tabs

Reply:
Looking at the Radix implementation, they still rely on the user-inputed `value` to create their full IDs. They feed the value into functions to create the trigger ID and the content ID, both based off of one `baseID` for the entire tab group. In their implementation, so if the user chose to input the same value for multiple tabs in the same group they'd run into the same issue. What their solution does address is having multiple tab groups on the same page, because the `baseID` will be different, and thus two tabs with value "Tab 1" will result in trigger ids: `{baseID}-trigger-Tab 1`, where the `baseID` will be different. 

Essentially, it looks they are guaranteeing that so long as you declare unique values for each tab in a tab list, you won't run into any issues, but I believe that there is still that dependance on the user entering unique values. 

![Screenshot 2024-06-04 at 12 16 54 PM](https://github.com/aws-amplify/amplify-ui/assets/170576352/b0aa75da-c3e4-4b5d-b16e-fefe1551e96a)

Reply:
Right, the issue is not so much with tabs within the same tab group, but for multiple Tab components in the DOM at the same time that use the same tab names. Within the same tab group, I think it's reasonable that users should specify unique tab names. But for multiple sets of the same tab names on a page, that should be something we can build into this component by generating a unique ID (like the baseId that Radix generates) that is used to build out the ID string.

An example of this is our Docs site that sometimes uses multiple tab components on a page, like [the docs for adding Facebook login, Google login, etc](https://docs.amplify.aws/gen1/react/build-a-backend/auth/add-social-provider/).